init.rb

```diff
@@ -98,6 +100,9 @@ def display_dyno_type_and_costs(app_resp, formation_resp)
       }
     end
 
+    # in case of an app not yet released
```

Comment:
Sorry if this is a silly question, but what exactly does "app not yet released" in this context mean? If it's been pushed to Heroku, it is by definition released, no? Or am I missing the point entirely? :)